tools/webdriver/webdriver/error.py

```diff
@@ -213,6 +221,8 @@ def from_response(response):
     code = value["error"]
     message = value["message"]
     stack = value["stacktrace"] or None
+    # data is optional, and could even be an empty dict
+    data = value.get("data") or None
 
     cls = get(code)
     return cls(response.status, code, message, stacktrace=stack)
```

Comment:
Without that we won't set the data property:

```suggestion
    return cls(response.status, code, message, stacktrace=stack, data=data)
```

Reply:
Fixed!